moveit_simple/include/moveit_simple/moveit_simple.h

```diff
@@ -111,7 +111,7 @@ class Robot
    * @param time - time from start of trajectory to reach point
    * @return true if point successfullly added
    */
-  bool addTrajPoint(const std::string & traj_name, const std::string & point_name,
+  void addTrajPoint(const std::string & traj_name, const std::string & point_name,
```

Comment:
Please update header comments with the possible exceptions.  I believe the exceptions are added as stated [here](http://cs.swan.ac.uk/~csoliver/ok-sat-library/internet_html/doc/doc/Doxygen/1.7.6.1/html/commands.html#cmdexception)